app/src/main/java/pm/gnosis/heimdall/ui/security/unlock/UnlockActivity.kt

```diff
@@ -61,13 +81,43 @@ class UnlockActivity : BaseActivity() {
                 .flatMap { viewModel.unlock(layout_unlock_password_input.text.toString()) }
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribeForResult(onNext = ::onState, onError = ::onUnlockError)
+
         disposables += layout_unlock_password_input.textChanges()
                 .window(200, TimeUnit.MILLISECONDS)
                 .flatMapMaybe { it.lastElement() }
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe({
                     animateHandle(it.length * 15f, interpolator = DecelerateInterpolator())
                 }, Timber::e)
+
+        disposables += layout_unlock_switch_to_password.clicks()
```

Comment:
shouldn'T we stop listening for the fingerprint? right now I could still unlock it with fingerprint, right? (not that big of a deal for me)